src/controller/Start.java

```diff
@@ -1,54 +1,110 @@
 package controller;
 
-import java.awt.event.ActionEvent;
+import java.awt.Font;
+import java.awt.Color;
 import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+
 
+import javax.swing.SwingConstants;
 import javax.swing.JPanel;
 
 /**
  * 
  * Start.java Start ist ein Controller zur verwaltung von aktionen des start
  * screens.
  * 
- * @author Nina
+ * @author Nina, Niklas E.
  * @since 24-05-2021
  */
 public class Start implements Controller {
-
+	
 	Switcher switcher;
 	view.Start view;
-
+	
 	public Start(Switcher s) {
 		this.switcher = s;
 		this.init();
 	}
-
+	
 	/**
 	 * Initialisiert das Start view element
 	 */
 	public void init() {
 		this.view = new view.Start();
-		this.view.getFirst().setText("Start");
-		this.view.getFirst().addActionListener(new ActionListener() {
+		this.initTitleLabel();
+		this.initStartButton();
+		this.initOptionsButton();
+		this.initExitButton();
+	}
+	
+	/**
+	 * Initialisiert das Title-Label
+	 */
+	public void initTitleLabel() {
+		this.view.getTitleLabel().setText("Quiz Projekt24");
+		this.view.getTitleLabel().setHorizontalAlignment(SwingConstants.CENTER);
```

Comment:
Theoretisch würde man den gesamten "Style" also alles was mit dem Aussehen des Screens zu tun hat in den View machen und wirklich nur die Controller spezifischen Sachen im Controller lassen - also zum Beispiel Aktionen von Buttons oder die Beschiftung :) Damit musst du dann alles nur ins View kopieren und so könnte man die Views super gut wiederverwenden - was so nicht ganz so schnell möglich ist :)

Reply:
Dann würde aber wirklich alles ins View kommen bis auf den addActionListener()-Aufruf, auch **feste** Beschriftung würde ich dann dahin packen weil diese niemals geändert wird z.B. der Start-Button wird immer den Text "Start" haben und das wird sich niemals ändern, nicht so wie Question-Label, das immer mit einem Buttonklick geändert wird sei es der Start-Button zum Spielstart für die erste Frage oder eine Antwortauswahl, die richitg ist und zur nächsten Frage führt.

Also mir egal ist ja fast nur Copy&Paste, soll ich alles ins Views packen oder nicht?

Reply:
Das View Element ist für das Aussehen, Formatierung usw verantwortlich. Beschriftungen, Tabellen oder Bindings für Aktionen finden normal immer im Controller statt. 

Ob Dinge nie wieder geändert werden, kann man vorher leider nie wissen. :v:

Reply:
Ok, wird so gemacht.